### PR TITLE
chore: mention automatic config setup with Gradle

### PR DESCRIPTION
**Pull request by thymikee**

### CODE REVIEW

**Comment by Krizzu** [comment]
We can add note that `haul init` will do it for you and leave those notes there as well

File: docs/Configuring Your Project.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Usually, a React Native project has a run script phase that runs `react-native-x
 
 ## Integrating with Gradle
 
-If you're on React Native version >= 0.43, add the following in `android/app/build.gradle` somewhere before the `apply from: "../../node_modules/react-native/react.gradle"` statement:
+If you're on React Native version >= 0.43, `haul init` will add the following code in `android/app/build.gradle` somewhere before the `apply from: "../../node_modules/react-native/react.gradle"` statement for you:
```
Comment (thymikee) [inline docs/Configuring Your Project.md]: @satya164 @Krizzu how about now?

Comment (satya164) [reply docs/Configuring Your Project.md]: Maybe something like:

If you're on React Native version >= 0.43, run the following to automatically configure your gradle config to use haul:

```sh
haul init
```

If the automatic setup didn't work for you, you can manually add the following code in android/app/build.gradle somewhere before the apply from: "../../node_modules/react-native/react.gradle" statement for you:

**Comment by thymikee** [comment]
Should be better now

**Comment by Krizzu** [comment]
Looks good 👍

**Pull request merged**